src/main/java/com/puppycrawl/tools/checkstyle/checks/design/OneTopLevelClassCheck.java

```diff
@@ -52,32 +52,32 @@
  * An example of code with violations:
  * </p>
  * <pre>
- * public class Foo{
- *   //methods
+ * public class Foo {
+ *   // top-level class
```

Comment:
Please add the `OK` mark to make it clear as much as possible, for example:
`// ok, first top-level class`
`// violation, second top-level class`

Reply:
Fixed